ocs_ci/ocs/node.py

```diff
@@ -1771,13 +1799,42 @@ def get_node_zone():
     node_objs = get_nodes(node_type=constants.WORKER_MACHINE)
     node_zone_dict = dict()
     for node_obj in node_objs:
-        node_zone_dict[node_obj.name] = node_obj.data["metadata"]["labels"][
-            "failure-domain.beta.kubernetes.io/zone"
-        ]
+        node_zone_dict[node_obj.name] = get_node_zone(node_obj)
     log.info(f"node-zone dictionary {node_zone_dict}")
     return node_zone_dict
 
 
+def get_node_rack_zone(failure_domain, node_obj):
```

Comment:
I think it is better to rename it to get_node_rack_or_zone because otherwise, it could be misinterpreted as "get the node rack's zone"

Reply:
Fixed